.eslintrc.json

```diff
@@ -5,7 +5,7 @@
     "sourceType": "script"
   },
   "rules": {
-    "no-unused-vars": ["error", { "vars": "all", "args": "none" }],
+    "no-unused-vars": ["error", { "argsIgnorePattern": "^_" }],
```

Comment:
:+1: With this convention we can declare all available callback function parameters in controller scripts, even if we don't need them.